cdap-distributions/src/emr/install.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+#
+# Copyright © 2015 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+#
+# Install CDAP under hadoop user for EMR
+#
+
+# Fetch repo file for YUM
+curl http://repository.cask.co/centos/6/x86_64/cdap/3.0/cask.repo > /etc/yum.repos.d/cdap-3.0.repo
+rpm --import http://repository.cask.co/centos/6/x86_64/cdap/3.0/pubkey.gpg
+# Enable EPEL packages (req'd for nodejs)
+sed -i -e '0,/enabled=0/s//enabled=1/' /etc/yum.repos.d/epel.repo
+yum makecache
+# Install dependencies
+yum install --downloadonly chkconfig -y
+rpm --install --force /var/cache/yum/x86_64/latest/amzn-main/packages/chkconfig-*
+yum install gyp http-parser-devel redhat-rpm-config icu -y
+yum install nodejs nodejs-devel npm -y
+yum install -y cdap-{gateway,kafka,master,ui}
```

Comment:
At the risk of sounding picky: I would have the '-y' right after 'yum install' for every command or at the end (for every command).